core/examples/luigi-sample-angular/src/app/project/project.component.ts

```diff
@@ -57,6 +59,15 @@ export class ProjectComponent implements OnInit, OnDestroy {
     if (this.cudListener) {
       const removed = removeContextUpdateListener(this.cudListener);
     }
+    this.unsubscribeSplitViewHandlers();
+  }
+
+  private unsubscribeSplitViewHandlers() {
```

Comment:
I think it would make sense to offer this out of the box when split view close happens, then we can remove some code from this component and in the implementation, the code under `window.onunload` can be moved to the `closed.ok` listener ;)

Reply:
Discussed, we will do it as described above, since each time a new split view is created the user gets a new instance of `splitViewHandle`.

Reply:
done